src/index.js

```diff
@@ -1,21 +1,40 @@
-import _ from 'lodash';
-import printMe from './print.js';
+// import _ from 'lodash';
+// import printMe from './print.js';
```

Comment:
- [ ] Please kindly remove `commented code` from your `code base`, as it makes your `code base` dirty and less readable.

Reply:
> * [ ]  Please kindly remove `commented code` from your `code base`, as it makes your `code base` dirty and less readable.

Comments removed, thank you ;-)